toolbox/util/githubClient.go

```diff
@@ -336,9 +336,12 @@ func (g GithubClient) CreateAnnotatedTag(repo, tag, sha, msg string) error {
 
 // CreateReleaseUploadArchives creates a release given release tag and
 // upload all files in archiveDir as assets of this release
-func (g GithubClient) CreateReleaseUploadArchives(repo, releaseTag, archiveDir string) error {
+func (g GithubClient) CreateReleaseUploadArchives(repo, releaseTag, sha, archiveDir string) error {
```

Comment:
Also update the function comment to explain what `sha` is used for

Reply:
one